app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/CustomPlaybackTransportControlGlue.java

```diff
@@ -197,6 +273,15 @@ public void onCustomActionClicked(Action action, View view) {
         }
     }
 
+    public void setEndTime() {
+        long msLeft = playerAdapter.getDuration() - playerAdapter.getCurrentPosition();
+        Calendar ends = Calendar.getInstance();
+        ends.setTimeInMillis(ends.getTimeInMillis() + msLeft);
+        Context context = getContext();
+        if (context != null && mEndsText != null)
```

Comment:
These null checks are unnecessary

Reply:
mEndsText does need to be null checked as it appears the play state gets updated before we hook into the view holder function